### PR TITLE
feat(support-bundle): Add new parameter: `SUPPORT_BUNDLE_COLLECTOR` to execute specified support-bundle-kit collector

### PR DESCRIPTION
**Pull request by weizhe0422**
[Longhorn 5073](https://github.com/longhorn/longhorn/issues/5073)

Signed-off-by: Ray Chang <ray.chang@suse.com>

A New parameter: `SUPPORT_BUNDLE_COLLECTOR` is passed to support-bundle-kit to execute the specified collector script.  And the [support-bundle-kit PR](https://github.com/rancher/support-bundle-kit/pull/63) will also accept this new parameter and execute `collector-longhorn.sh` to collect system information and logs.
image
image

### CODE REVIEW

File: controller/support_bundle_controller.go [hunk 1]
```diff
@@ -683,6 +683,10 @@ func (c *SupportBundleController) newSupportBundleManager(supportBundle *longhor
 									Name:  "SUPPORT_BUNDLE_REGISTRY_SECRET",
 									Value: string(registrySecret),
 								},
+								{
+									Name:  "SPECIFY_COLLECTOR_SCRIPT",
```
Comment (innobead) [inline controller/support_bundle_controller.go:687]: Suggest having the same pattern like `SUPPORT_BUNDEL_COLLECTOR` instead.

**Review by innobead — CHANGES_REQUESTED** [review]

**Comment by weizhe0422** [comment]
@Mergifyio backport v1.4.x v1.3.x

**Comment by weizhe0422** [comment]
The backport to v1.3.x [PR#1756 https://github.com/longhorn/longhorn-manager/pull/1756] was closed since the support bundle kit has only been introduced since v1.4.x.

**Comment by innobead** [comment]
@weizhe0422 this is no `backport/v1.3.x` for this issue. Please just follow the backport labels on that issue to do the backport instead.

**Pull request merged**